frontend/talentsearch/src/js/lang/fr.json

```diff
@@ -1496,24 +1496,24 @@
     "description": "Legend for Skills filter checklist"
   },
-  "E1WElo": {
-    "defaultMessage": "Les personnes dont le sexe est déclaré sont des femmes, en tout ou en partie. Cela comprend les femmes cisgenres (cis) et transgenres (trans).",
-    "description": "Definition of accepted ways to identify as a women"
+  "6danS7": {
+    "defaultMessage": "Cette catégorie comprend les personnes dont le genre déclaré est féminin. Elle comprend les femmes cisgenres (cis) et les femmes transgenres (trans).",
+    "description": "Definition of the Woman category from the StatsCan 'Classification of gender' page."
   },
   "Jtz3yb": {
     "defaultMessage": "Je suis autochtone.",
     "description": "Label for the checkbox to identify as indigenous under employment equity"
   },
-  "NFL+q8": {
-    "defaultMessage": "Personnes appartenant à une minorité visible, autres que les autochtones, qui ne sont pas de race blanche ou qui n'ont pas la peau blanche. Voici des exemples de groupes de minorités visibles : Sud-Asiatique, Chinois, Noir, Philippin, Arabe, Latino-Américain, Asiatique du Sud-Est, Asiatique occidental, Coréen et Japonais.",
-    "description": "Definition of accepted ways to identify as a visible minority"
+  "F4K5RB": {
+    "defaultMessage": "Minorité visible réfère au fait qu'une personne est ou non une minorité visible, tel que défini dans la Loi sur l'équité en matière d'emploi. Dans le cadre de la Loi sur l'équité en matière d'emploi, les minorités visibles sont définies comme « les personnes, autres que les Autochtones, qui ne sont pas de race blanche ou qui n'ont pas la peau blanche ». La population des minorités visibles est principalement composée des groupes suivants : Sud-Asiatique, Chinois, Noir, Philippin, Arabe, Latino-Américain, Asiatique du Sud-Est, Asiatique occidental, Coréen et Japonais.",
+    "description": "Definition of Visible minority from the StatsCan 'Visible minority of person' page."
   },
   "bKvvaI": {
     "defaultMessage": "Conditions d'emploi",
     "description": "Legend for the Conditions of Employment filter checklist"
   },
-  "hz8OYx": {
-    "defaultMessage": "Les personnes ayant une déficience, y compris une déficience physique, intellectuelle, cognitive, mentale ou sensorielle, un trouble d'apprentissage ou de la communication ou une limitation fonctionnelle, de nature permanente, temporaire ou épisodique, manifeste ou non et dont l'interaction avec un obstacle nuit à la participation pleine et égale d'une personne dans la société.",
+  "y5Z2Li": {
+    "defaultMessage": "Réfère à une personne dont les activités quotidiennes sont limitées en raison d'un trouble ou d'une difficulté à accomplir certaines tâches. La seule exception à cet égard concerne les troubles du développement, le répondant qui a reçu un tel diagnostic étant considéré comme ayant une incapacité.",
     "description": "Definition of accepted ways to identify as person with a disability."
```

Comment:
```suggestion
    "description": "Definition of Person with a disability from the StatsCan 'Classification of Status of Disability' page."
```

Reply:
Good catch, thanks!

Fixed: [Fix description of disabled translation](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4487/commits/75860cad80a1b26e3490e5bae9b106c257855d53)